src/components/Articles/LikeArticle/LikeArticle.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp, faThumbsDown } from '@fortawesome/free-solid-svg-icons';
+import PropTypes from 'prop-types';
+import { getArticleLikes, likeArticle, dislikeArticle } from '../../../actions';
+import { Button } from '../../common';
+import { clearHighlightArticleStore } from '../../../actions/articles';
+import './LikeArticle.scss';
+
+const style = { color: '' };
```

Comment:
Why do you have a style object here?

Reply:
Remove this style object if you are not using it